maxim/tests/test_connection_retry_logic.py

```
@@ -0,0 +1,713 @@
+    ReadTimeout,
+    RequestError,
+    TimeoutException,
+)
+
+# Add the maxim package to the path
+import sys
+import os
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "."))
+
+from maxim.apis.maxim_apis import MaximAPI, ConnectionPool
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)